packages/roosterjs-content-model-plugins/lib/imageEdit/utils/findEditingImage.ts

```diff
@@ -20,7 +20,7 @@ export function findEditingImage(
             for (const segment of paragraph.segments) {
                 if (
                     segment.segmentType == 'Image' &&
-                    ((imageId && segment.format.id == imageId) || segment.dataset.isEditing)
+                    ((imageId && segment.format.id == imageId) || segment.format.imageMarker)
```

Comment:
So this means as long as image has a marker, no matter what value it is, we treat it as editing?

Reply:
In that case, you can declare it as a boolean. So we treat it as editing when its value is true.

Reply:
I changed to verify if the marker equal to isEditing, maybe as string we can reuse this marker to point order states than editing.

Reply:
That is a good idea. So let's rename it to "state" instead of "marker" since marker is too common and not easy to understand what it is used for.